Vagrantfile

```diff
@@ -14,6 +14,10 @@ guest_config_dir = ENV['DRUPALVM_CONFIG_DIR'] ? "/vagrant/#{ENV['DRUPALVM_CONFIG
 
 drupalvm_env = ENV['DRUPALVM_ENV'] || 'vagrant'
 
+host_os = %w[darwin bsd freebsd linux solaris windows].reduce(nil) do |detected_os, os|
```

Comment:
Vagrant::Util::Platform also has a `cygwin?` method, but that is probably not needed. Need to ensure that `windows?` returns true for cygwin.